frontend/mulighetsrommet-veileder-flate/src/components/filtrering/CheckboxFilter.tsx

```diff
@@ -1,10 +1,12 @@
 import { Accordion, Alert, Checkbox, CheckboxGroup } from "@navikt/ds-react";
 import { useAtom } from "jotai";
 import React from "react";
-import { filterAccordionAtom } from "../../core/atoms/atoms";
-import { addOrRemove, kebabCase } from "../../utils/Utils";
+import { filterAccordionAtom } from "@/core/atoms/atoms";
+import { addOrRemove, kebabCase } from "@/utils/Utils";
+import styles from "@/components/filtrering/NavEnhetFilter.module.scss";
```

Comment:
Burde denne scss-filen hatt navnet `CheckboxFilter` i stedet for `NavEnhetFilter`, eller kunne klassen `accordion_header_med_antall` blitt inkludert i react-komponenten som sendes inn som del av  `accordionHeader` prop i stedet? 
(Henger meg bare litt opp i at CheckboxFilter høres ut som en generell komponent, men at den ikke virker veldig generell når den trekkes inn styles rettet mot NavEnhet.)